src/ucp/core/ucp_worker.c

```diff
@@ -1957,14 +1951,26 @@ ucs_thread_mode_t ucp_worker_get_thread_mode(uint64_t worker_flags)
 
 static void ucp_warn_unused_uct_config(ucp_context_h context)
 {
+    int num_unused_cached_kv = 0;
+    ucs_string_buffer_t unused_cached_uct_cfg;
     ucs_config_cached_key_t *key_val;
 
+    ucs_string_buffer_init(&unused_cached_uct_cfg);
```

Comment:
pls do ucs_string_buffer_cleanup at the end to release memory

Reply:
Done